frontend/src/Response/Table/BaseTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from "@mui/material";
+import { WithInputInfoInterface } from "../SimpleQuestions/LabeledQuestion";
+
+
+export interface BaseTableInterface extends WithInputInfoInterface {
+    rows: Array<Array<JSX.Element>>;
+}
+
+export enum TableType {
+    FIXED_TABLE = "FIXED_TABLE",
+    DYNAMIC_TABLE = "DYNAMIC_TABLE",
+}
+
+
+function BaseTable(baseTableData: BaseTableInterface): JSX.Element {
```

Comment:
May be my comments about width of this table refer to this file - I think that table's width must depends on number of columns and if this number is too big table must be scrollable